File: python/templates/outbound-ops/app/brief.py
"""Morning Brief (CLI) — a thin wrapper over the OutboundOps class.

The logic lives in app/ops.py so you can also use it in your own service:

    from app.ops import OutboundOps
    brief = OutboundOps(client).brief()
"""
import json
from datetime import datetime, timezone

from .ops import OutboundOps
from .render import render_brief


def run(argv=None) -> None:
    argv = argv or []
    raw = "--raw" in argv
    result = OutboundOps.from_env().brief()
    now = datetime.now(timezone.utc).strftime("%Y-%m-%d %H:%M") + " UTC"
    print(render_brief(now, result["health"], result["replies"], result["overview"], result["actions"]))
    if raw:
        print("\n--- raw ---\n" + json.dumps(result, indent=2))

File: js/templates/outbound-ops/commands/brief.ts
/**
 * Morning Brief (CLI) — a thin wrapper over the OutboundOps class.
 * The logic lives in lib/ops.ts so you can also use it in your own service:
 *
 *   import { OutboundOps } from "../lib/ops.js"
 *   const brief = await new OutboundOps(client).brief()
 *
 * Run:  INSTANTLY_API_KEY=... npm run brief   [-- --raw]
 */
import { InstantlyApiError } from "@instantly-ai/sdk"
import { OutboundOps } from "../lib/ops.js"
import { renderBrief } from "../lib/render.js"

async function main(): Promise<void> {
  const raw = process.argv.includes("--raw")
  const result = await OutboundOps.fromEnv().brief()
  const now = new Date().toISOString().replace("T", " ").slice(0, 16) + " UTC"
  console.log(renderBrief({ now, ...result }))
  if (raw) console.log("\n--- raw ---\n" + JSON.stringify(result, null, 2))
}

main().catch((err) => {
  if (err instanceof InstantlyApiError) {
    console.error(`✗ API error ${err.status}:`, err.payload)
    process.exit(1)
  }
  console.error(err instanceof Error ? err.message : err)
  process.exit(1)
})
